SummerProject/middleware/user_auth.py

```diff
@@ -0,0 +1,19 @@
+# pylint: disable=R0205,R1710,R0201
+
+"""Custom middleware"""
+from django.http.response import HttpResponseBadRequest
+
+
+class AuthenticatedUser(object):
```

Comment:
It's a good practice to include `Middleware` to the middleware class name. Eg: `AuthenticatedUserMiddleware`